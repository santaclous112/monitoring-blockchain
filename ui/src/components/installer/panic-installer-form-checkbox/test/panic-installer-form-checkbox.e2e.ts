import { newE2EPage } from '@stencil/core/testing';

describe('panic-installer-form-checkbox', () => {
  it('render the inputs in the correct order', async () => {
    const page = await newE2EPage();
    await page.setContent('<panic-installer-form-checkbox></panic-installer-form-checkbox>');

    const element = await page.find('panic-installer-form-checkbox');
    expect(element).toMatchSnapshot();
  });
});