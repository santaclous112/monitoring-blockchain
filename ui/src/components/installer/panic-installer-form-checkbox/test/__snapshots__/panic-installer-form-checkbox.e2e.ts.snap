// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`panic-installer-form-checkbox render the inputs in the correct order 1`] = `
<panic-installer-form-checkbox class="hydrated">
  <div>
    <input type="hidden" value="false">
    <input type="checkbox" value="true">
  </div>
</panic-installer-form-checkbox>
`;
